unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/NFTUtils/NFTUtils.cs

```diff
@@ -143,17 +142,30 @@ public static bool TryParseUrn(string urn, out string chain, out string contract
                 var contractStandardSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
                 urnSpan = urnSpan.Slice(contractStandardSpan.Length + 1);
 
-                // 4: contract address
-                var contractAddressSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
-                urnSpan = urnSpan.Slice(contractAddressSpan.Length + 1);
-
-                // 5: token id
-                var tokenIdSpan = urnSpan;
-                contractAddress = contractAddressSpan.ToString();
-                tokenId = tokenIdSpan.ToString();
-                chain = chainSpan.ToString();
-
-                return true;
+                // Check if the URN contains "collections-thirdparty"
+                if (contractStandardSpan.ToString().Equals(COLLECTIONS_THIRDPARTY, StringComparison.Ordinal))
```

Comment:
I'm not too familiar with this method and how/where it's used, but I'm concerned that in this block we're not setting a new value for `tokenId`, is that expected for third-party collections? Can we add a note stating that for future reference?

Reply:
That is correct yes, Third-Party wearables do not have token id at the moment. I'll add a comment.